docs/01_nodeos/03_plugins/chain_plugin/index.md

```diff
@@ -120,8 +120,8 @@ Config Options for eosio::chain_plugin:
 										available cores.  
   --rocksdb-files arg (=-1)             Max number of rocksdb files to keep 
                                         open. -1 = unlimited.
-  --rocksdb-write-buffer-size-mb arg (=134217728)
-                                        Size of a single rocksdb memtable
+  --rocksdb-write-buffer-size-mb arg (=128MB)
+                                        Size of a single rocksdb memtable (in MiB)
```

Comment:
```
  --rocksdb-write-buffer-size-mb arg (=128)
                                        Size of a single rocksdb memtable (in MiB)
```
If you specify `128MB` on the command line that will not work.

Reply:
Removed MB from the doc, now only in description.

Reply:
Looks good passing just the 128 without the MB.

Reply:
updated